client/src/visualiser-debugger/Component/Visualizer/Entities/DrawableEntities/drawablePointer.tsx

```diff
@@ -67,6 +81,22 @@ const PointerDrawable: DrawablePointerComponent = (
     setCoords(res);
   }, [pos.x.val, pos.y.val]);
 
+  const splitLabels = (label: string) => {
+    return label.split(', ');
+  };
+
+  // Calculate the text's width using canvas
+  // https://www.tutorialspoint.com/Calculate-text-width-with-JavaScript#:~:text=To%20calculate%20text%20width%2C%20we,method%20to%20measure%20the%20text.
+  const getTextWidth = (text: string, fontSize: number) => {
```

Comment:
We can do 

```
const canvas = document.createElement('canvas');
const context = canvas.getContext('2d');

const getTextWidth = (text: string, fontSize: number) => {
  if (context) {
    context.font = `${fontSize}px Arial`;
    return context.measureText(text).width;
  }
  return 0;
};
```

To avoid create this canvas element everytime, and we only need an estimation, so I am not sure if creates a canvas to do so is the best choice.

Reply:
Do you have any suggestions for other alternatives? I've tried to use the arrow's coordinate, but it points to the start of the label instead

Reply:
You know what I think it's fine